__tests__/commit-file-async.js

```diff
@@ -6,14 +6,19 @@ const path = require('path');
 const memFs = require('mem-fs');
 const sinon = require('sinon');
 const editor = require('..');
+const { STATE, STATE_MODIFIED, STATE_DELETED } = require('../lib/state');
 
 const rmSync = filesystem.rmSync || filesystem.rmdirSync;
 
-describe('#commit()', () => {
-  const outputRoot = path.join(os.tmpdir(), 'mem-fs-editor-test');
+// Permission mode are handled differently by windows, skip it.
+const posixIt = process.platform === 'win32' ? it.skip : it;
```

Comment:
I'd recommend adding windows to the matrix; seems like it'd bring value for this package specifically :)

Reply:
Not sure if this was clear, but I meant in the github action; so we run the UTs in windows.

Reply:
It’s already running.
Or I am missing something?

Reply:
oh lol 🤦🏻 you're right, this is already the case!